components/post_view/post_list_virtualized/post_list_virtualized.jsx

```diff
@@ -118,18 +132,20 @@ class PostList extends React.PureComponent {
         this.state = {
             isScrolling: false,
             isMobile,
-            atBottom: false,
+            atBottom: true,
```

Comment:
If this always starts as true, does that mean the toast will always appear when this component is first mounted?

Reply:
it's been a long time, but i believe this was because of how it behave when mounting, sudheer might confirm this

Reply:
Kinda of opposite, It starts with `true` now so the toast is not shown on mount if it is not needed. Otherwise i have seen flickers of this at times

Reply:
Reverted this as i ended up changing the way things are structured.